elasticdl/pkg/ps/server_test.go

```diff
@@ -218,38 +199,52 @@ func TestPullEmbeddingVector(t *testing.T) {
 	defer conn.Close()
 	defer cancel()
 
-	var request1 pb.Model
-	// embedding table info
-	var epb pb.EmbeddingTableInfo
-	epb.Name = "e1"
-	epb.Dim = 2
-	epb.Initializer = "zero"
-	request1.EmbeddingTableInfo = append(request1.EmbeddingTableInfo, &epb)
+	var request = &proto.Model{
+		DenseParameters: make(map[string]*tensor_go_proto.TensorProto),
+		EmbeddingTables: make(map[string]*proto.IndexedSlicesProto),
+		EmbeddingTableInfo: []*proto.EmbeddingTableInfo{&proto.EmbeddingTableInfo{
+			Name:        "e1",
+			Dim:         10,
+			Initializer: "zero",
+			Dtype:       commonnew.Float32,
+		}},
+	}
 
-	_, err1 := client.PushEmbeddingInfo(ctx, &request1)
-	if err1 != nil {
-		t.Errorf("Failed to push embedding vector info")
+	// dense embedding param
+	a := make([]float32, 10)
+	b := make([]float32, 10)
+	c := make([]float32, 10)
+	for i := 0; i < 10; i++ {
+		a[i] = rand.Float32()
+		b[i] = rand.Float32()
+		c[i] = rand.Float32()
 	}
+	d := []int64{2, 5}
+	t1 := commonnew.NewTensor(a, d) // t1
+	t2 := commonnew.NewTensor(b, d) // t2
 
-	var request2 pb.PullEmbeddingVectorRequest
-	ids := []int64{1, 3, 5}
-	request2.Name = "e1"
-	request2.Ids = ids
+	ed := []int64{1, 10}
+	e1 := commonnew.NewIndexedSlices(commonnew.NewTensor(c, ed), []int64{1})
 
-	res, err2 := client.PullEmbeddingVector(ctx, &request2)
-	if err2 != nil {
-		t.Errorf("Failed to pull embedding vector")
+	request.DenseParameters["t1"] = t1.SerializeToTensorProto()
+	request.DenseParameters["t2"] = t2.SerializeToTensorProto()
+	request.EmbeddingTables["e1"] = e1.SerializeToIndexedSlicesProto()
+
+	client.PushModel(ctx, request)
+
+	pr := &proto.PullDenseParametersRequest{
+		Version: 0,
 	}
 
-	assert.Contains(t, s.Param.EmbeddingParam, "e1")
-	tensor := common.DeserializeTensorPB(res)
-	assert.Equal(t, "e1", tensor.Name)
-	assert.Equal(t, ids, tensor.Indices)
-	assert.Equal(t, 6, len(tensor.Value))
+	resp, _ := client.PullDenseParameters(ctx, pr)
+	assert.Equal(t, true, resp.Initialized)
+	assert.Equal(t, int32(0), resp.Version)
+	assert.True(t, commonnew.CompareFloatArray(a, commonnew.Slice(commonnew.DeserializeFromTensorProto(resp.DenseParameters["t1"])).([]float32), 0.0001))
+	assert.True(t, commonnew.CompareFloatArray(b, commonnew.Slice(commonnew.DeserializeFromTensorProto(resp.DenseParameters["t2"])).([]float32), 0.0001))
 	gs.Stop()
 }
 
-func TestPushGradients(t *testing.T) {
+func TestPushGradient(t *testing.T) {
```

Comment:
`TestPushGradient` --> `TestPushGradients`. Since it's the test for RPC `PushGradients`

Reply:
fixed